frontend/integration-tests/protractor.conf.ts

```diff
@@ -213,3 +213,10 @@ export const waitForNone = (elementArrayFinder) => {
     return count === 0;
   };
 };
+
+export const create = (obj) => {
```

Comment:
Was experiencing issues with `echo <some JSON> | kubectl create -f -`, so use a temporary file with cleanup instead.

Reply:
Does this get created in the top-level console dir? It might be better to put this in a tmp folder or in a folder that in our gitignore. If create fails, I'm guessing the file doesn't get deleted.